app/assets/stylesheets/simulator.scss

```diff
@@ -1,3 +1,19 @@
 // Place all the styles related to the simulator controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+$gray-dark: #b9b9b9;
+$blue: #337ab7;
+$primary: $gray-dark;
+$secondary: $blue;
+
+@import 'bootstrap/scss/bootstrap';
+
+.alert {
+  width: 100%;
+}
```

Comment:
Might not be needed custom class, maybe you can use the bootstrap class itself https://getbootstrap.com/docs/4.0/utilities/sizing/. check this out